### ISSUE TITLE
Is this needed anymore with Next 11.1?

### ISSUE THREAD

**Issue by DustinBrett** [FormidableLabs/jest-next-dynamic#34]
I removed it and it worked without it. Based on seeing this in the release notes:

https://github.com/vercel/next.js/pull/26614

**Comment by leerob** [comment]
Correct, this library can now be deprecated! Thank you so much to the Formidable team for helping patch this issue in the interim 🙌

**Comment by DustinBrett** [comment]
I recently needed this again with 12.0.8 of next.js.

**Comment by buesing** [comment]
@leerob do we have an official info, is this necessary or not from v12 on?

**Comment by DustinBrett** [comment]
I've since gone to using `next/jest`. (https://nextjs.org/docs/testing)

**Comment by buesing** [comment]
It's still happening though, even with the app from their examples. Made a new issue in next: https://github.com/vercel/next.js/issues/41725

**Comment by DustinBrett** [comment]
Ah ok darn. I don't even think I do the dynamic import test anymore. Perhaps I gave up.
